src/workers/parser/utils/formatSymbols.js

```diff
@@ -23,7 +23,10 @@ export function formatSymbols(source: Source) {
     const expression = symbol.expression || "";
     const klass = symbol.klass || "";
 
-    return `${loc} ${exprLoc} ${expression} ${symbol.name}${params} ${klass}`.trim(); // eslint-disable-line max-len
+    const name = symbol.name || "";
+    const names = (symbol.specifiers || []).join(", ");
```

Comment:
to avoid garbage array creation `const names = symbol.specifiers ? symbol.specifiers.join(", ") : "";`

Reply:
haha - yeah, that makes sense. `format` is just for testing